tracing/config/tracing.go

```diff
@@ -127,6 +128,9 @@ func NewTracingConfigFromMap(cfgMap map[string]string) (*Config, error) {
 
 // NewTracingConfigFromConfigMap returns a Config for the given configmap
 func NewTracingConfigFromConfigMap(config *corev1.ConfigMap) (*Config, error) {
+	if config == nil {
+		return NewTracingConfigFromMap(map[string]string{})
```

Comment:
```suggestion
		return NewTracingConfigFromMap(nil)
```

Reply:
Not sure this works

Reply:
It should since there are only read operations on the map, which work for `nil` maps.

Reply:
I think that function needs a default value in there

Reply:
I wasn't aware of that, thanks for the tip!

Reply:
This 100% works :)